### PR TITLE
refactor toolbutton.tsx - add default props

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

File: src/components/ToolButton.tsx [hunk 1]
```diff
@@ -48,9 +48,9 @@ export const ToolButton = React.forwardRef(function (
     return (
       <button
         className={`ToolIcon_type_button ToolIcon ${sizeCn}${
-          props.selected ? " ToolIcon--selected" : ""
-        } ${props.className || ""} ${
-          props.visible || props.visible == null
+          props.selected && " ToolIcon--selected"
```
Comment (dwelle) [inline src/components/ToolButton.tsx:51]: Can't do this lest `selected` is false or `undefined`

Comment (ad1992) [reply src/components/ToolButton.tsx:51]: didn't get you... The `ToolIcon--selected` should only be added when `selected` is true and that's what happening here. When selected is not true, `false` gets added.

Comment (dwelle) [reply src/components/ToolButton.tsx:51]: The main problem here is that there's no space between `${sizeCn}` and the next expression, so you'll get `ToolIcon_size_mundefined` class names. Besides, why do you want to add `false`/`undefined` as a className (and thus make a DOM mutation) when you can avoid it using the ternary as we did before?

Comment (ad1992) [reply src/components/ToolButton.tsx]: Agreed 👍 
I have reverted the change. Btw we should integrate https://github.com/JedWatson/classname
thoughts?

Comment (vjeux) [reply src/components/ToolButton.tsx]: Let's not until we find more use cases.

Comment (ad1992) [reply src/components/ToolButton.tsx]: @vjeux I think we have good number of use cases, the most recent was when adding zen mode class conditionally. So maybe we should check the overall codebase once and give it a thought?

**Pull request merged**